src/shared/utils/userJoined.ts

```diff
@@ -1,16 +1,19 @@
-import { GuildMember } from "discord.js";
+import { Client, GuildMember } from "discord.js";
 import * as db from "../../db/db";
 import { verifyUser } from "../../commands/verify/subcommands/util";
 import { isDangerOfNollan } from "./hodis";
 import { isDarkmode } from "./darkmode";
 
-export const userJoined = async (member: GuildMember): Promise<void> => {
+export const userJoined = async (
+	member: GuildMember,
+	client: Client
```

Comment:
It's kind of cursed to pass this entire Client around instead of a boolean `isLight` or a `ClientType` enum, but at this point whatever

Reply:
Open an issue if you want, I'm merging anyway because the bot keeps crashing

Reply:
cc @Benjaneb idk if it notifies you